data/assets/examples/navigationstate.asset

```diff
@@ -0,0 +1,34 @@
+-- Basic
+-- Example of using getNavigationState and setNavigationState to manipulate
```

Comment:
Maybe mention that this is an action, so users know where to look for it

Reply:
I added to the comment to say this is an action